### ISSUE TITLE
Rename `_komodo.pth` to `zzz-komodo.pth`

### ISSUE THREAD

**Issue by pinkwah** [equinor/komodoenv#44]
If one pip installs with `--editable`, doing `pywhich X` will point to the old version of the package rather than the new.

Renaming the file to something that comes late alphabetically will mean it gets found last, and thus any overridden package is found earlier.

**Comment by xjules** [comment]
Tested it (while installing ert with -e) and it seems to be still an issue.

**Comment by pinkwah** [comment]
Yes. I have tried different things and none of them are reproducible. I am half-convinced that in order for `komodoenv` to live up to its promise we'd need to implement our own `site.py` (ie, the Python module that does the resolving of site packages)

Not sure though.
